bin/katello-certs-check

```diff
@@ -157,7 +157,7 @@ function check-priv-key () {
 function check-ca-bundle () {
     printf "Checking CA bundle against the certificate file: "
     ERROR_PATTERN="error [0-9]+ at"
-    CHECK=$(openssl verify -CAfile $CA_BUNDLE_FILE -purpose sslserver -verbose $CERT_FILE 2>&1)
+    CHECK=$(openssl verify -no-CApath -CAfile $CA_BUNDLE_FILE -purpose sslserver -verbose $CERT_FILE 2>&1)
```

Comment:
today I learned (you do not want to know how), that it's sometimes required to also add `-no-CAstore` here

otherwise, at least on Debian with `ca-certificates` installed `/usr/lib/ssl/certs` is still consulted.